Sources/ApplicativeRouter/Combinators.swift

```diff
@@ -125,30 +133,28 @@ extension Router {
 }
 
 /// Parses the query params to create a value of type `A` via the `Codable` protocol.
-/// TODO: this only works for types `A` with string fields. Is it possible to improve that?
```

Comment:
`queryParams` is now fully codable! We can give it a type that has non-string fields!